index.js

```
@@ -214,6 +213,15 @@ export default class Supercluster {
         return expansionZoom;
     }
 
+    updatePointProperties(point) {
+        const id = this._linearSearchInPoints(point);
+        if (!id) return;
```

maybe we should also throw an error in this case?